src/ORM/DataQuery.php

```diff
@@ -752,12 +752,8 @@ public function reverseSort()
 
     /**
      * Set the limit of this query.
-     *
-     * @param int $limit
-     * @param int $offset
-     * @return $this
      */
-    public function limit($limit, $offset = 0)
+    public function limit(?int $limit, int $offset = 0): static
     {
         $this->query->setLimit($limit, $offset);
```

Comment:
No exceptions on this one?

Reply:
This method is really just an Alias for calling `$query->setLimit()`. So it seems rational to just let `setLimit` do all the work.